index.js

```diff
@@ -88,6 +88,10 @@ is.empty = function (value) {
     return true;
   }
 
+  if (is.nil(value)) {
```

Comment:
I'd be down for that, just wasn't sure how far-reaching my change should be. I'll make that change right now.